check.go

```diff
@@ -272,10 +278,16 @@ type CheckServiceConfigurationUpdateInput struct {
 
 type CheckServiceOwnershipCreateInput struct {
 	CheckCreateInput
+
+	RequireContactMethod bool         `graphql:"requireContactMethod"`
```

Comment:
these need to be `json` struct tags instead of `graphql` struct tags

```
RequireContactMethod bool         `json:"requireContactMethod"`
```

Also are these both "optional" fields?  Theres some ticky tacky ness with our graphql framework and nullability.

Reply:
If they are both "optional" fields (nullable) then it needs to be

```
	RequireContactMethod *bool         `json:"requireContactMethod,omitempty"`
	ContactMethod        *ContactType `json:"contactMethod,omitempty"`
```

Reply:
What does the asterisk before the type and the `omitempty` do? I assumed just `omitempty` was needed.